recipes/pysimplegui/yum_requirements.txt

```diff
@@ -0,0 +1 @@
+mesa-libGL
```

Comment:
You should confirm that you need mesa-libGL for this package.

Reply:
Prior to adding to as a yum-requirement, I got the following error during testing:

ImportError: libX11.so.6: cannot open shared object file

for linux builds. After searching for a solution, I found the following thread:

https://github.com/conda-forge/itkwidgets-feedstock/pull/6

which resolved the issue.